hack/csi_test/cmd/main.go

```diff
@@ -9,8 +9,8 @@ import (
 
 var (
 	endpoint    = flag.String("endpoint", "/tmp/csi.sock", "CSI endpoint")
-	host        = flag.String("host", "localhost", "Host machine name")
-	volume      = flag.String("volume", "", "Volume name")
+	shareAddr   = flag.String("shareAddr", "localhost", "Address of the share")
+	shareName   = flag.String("volume", "", "Volume name")
 	targetPath  = flag.String("targetPath", "", "Path to mount volume at")
 	request     = flag.String("request", "publish", "CSI request type")
```

Comment:
make the options nodePublish and nodeUnpublish to align more with the CSI spec and disambiguate from controllerPublish/Unpublish

Reply:
Also define them as consts since we use them in multiple places